### DISCUSSION TITLE
Bottom sheet modal stuck on the top after the keyboards shows

### DISCUSSION THREAD

**Discussion by nimeshmaharjan1** [gorhom/react-native-bottom-sheet#1790] [Q&A]
While using the BottomSheetTextInput after the input has been focused the sheet expands but after unfocusing it is stuck there any solutions for this?

https://github.com/gorhom/react-native-bottom-sheet/assets/86953478/05bdbf5d-bf72-468b-a2c6-2cd180677816

**Comment by ryanengland** [comment]
I think you're looking for the `keyboardBlurBehaviour` and the 'restore' option, that should resize the modal back to correct size.
